PhoneBook/src/main/java/humphrey/jeriah/PhoneBook.java

```diff
@@ -0,0 +1,52 @@
+package humphrey.jeriah;
+import java.util.*;
+
+/**
+ * Created by jeriahhumphrey on 2/7/17.
+ */
+public class PhoneBook {
+
+     TreeMap <String, String> phoneBook = new TreeMap();
+
+
+     public String lookUp(String key){
+         return phoneBook.get(key);
+     }
+
+     public void addEntry (String name, String number){
+         phoneBook.put(name, number);
+
+     }
+
+     public TreeMap getMap(){
```

Comment:
Yeah. getmap was for testing because I couldn't access the private map in the test class